src/main/java/com/minecolonies/coremod/commands/ActionMenu.java

```diff
@@ -63,4 +57,72 @@ public ForgePermissionNodes getForgePermissionNode()
     {
         return actionCommandClass;
     }
+
+    @Override
+    public int hashCode()
+    {
+        final int prime = 31;
+        int result = 1;
+        result = prime * result + ((actionArgumentList == null) ? 0 : actionArgumentList.hashCode());
+        result = prime * result + ((actionCommandClass == null) ? 0 : actionCommandClass.hashCode());
+        result = prime * result + ((description == null) ? 0 : description.hashCode());
+        result = prime * result + ((forgePermissionNode == null) ? 0 : forgePermissionNode.hashCode());
+        return result;
+    }
+
+    @Override
+    public boolean equals(final Object obj)
+    {
+        if (this == obj)
+        {
+            return true;
+        }
+        if (obj == null)
+        {
+            return false;
+        }
+        if (getClass() != obj.getClass())
+        {
+            return false;
+        }
+        final ActionMenu other = (ActionMenu) obj;
+        if (actionArgumentList == null)
+        {
+            if (other.actionArgumentList != null)
+            {
+                return false;
+            }
+        }
+        else if (!actionArgumentList.equals(other.actionArgumentList))
+        {
+            return false;
+        }
+        if (actionCommandClass != other.actionCommandClass)
+        {
+            return false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 13. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AIndentationCheck)